docs/en/ingest-management/troubleshooting/faq.asciidoc

```diff
@@ -165,12 +165,12 @@ server or host your own {package-registry}. To learn more, refer to
 [[does-agent-download-anything-from-internet]]
 == Does {agent} download anything from the Internet?
 
-* In version 7.10 and later, the endpoint security capabilities are bundled with {agent},
-so a fully capable artifact can be installed with no connection to the Elastic
-download site.  However, if it is in use, the {elastic-endpoint-integration} process
-is instructed to attempt to download newer released versions of the integration specific 
-artifacts it uses.  Some of those are, for example, the malware model, trusted apps artifact, 
-exceptions list artifact, and others.  For more information, see the 
+* In version 7.10 and later, a fully capable artifact can be installed with no
+connection to the Elastic download site.  However, if it is in use, the
+{endpoint-cloud-sec} process is instructed to attempt to download
+newer released versions of the integration specific  artifacts it uses.  Some of
+those are, for example, the malware model, trusted applications artifact, exceptions
+list artifact, and others.  For more information, see the 
```

Comment:
👍 If Endpoint cannot download these updates it will still protect the host. It just won't receive updates to protections without upgrading the Agent (and hence Endpoint) versions.